[2024-05-05 09:50:25,362] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.building_db manual__2024-05-05T09:50:20.624720+00:00 [queued]>
[2024-05-05 09:50:25,366] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.building_db manual__2024-05-05T09:50:20.624720+00:00 [queued]>
[2024-05-05 09:50:25,366] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 09:50:25,366] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-05-05 09:50:25,367] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 09:50:25,389] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): building_db> on 2024-05-05 09:50:20.624720+00:00
[2024-05-05 09:50:25,392] {standard_task_runner.py:52} INFO - Started process 1102 to run task
[2024-05-05 09:50:25,395] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'building_db', 'manual__2024-05-05T09:50:20.624720+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/.ipynb_checkpoints/lead_scoring_data_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmp60t4jp7p', '--error-file', '/tmp/tmp4nuwt79i']
[2024-05-05 09:50:25,397] {standard_task_runner.py:80} INFO - Job 3: Subtask building_db
[2024-05-05 09:50:25,441] {task_command.py:371} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.building_db manual__2024-05-05T09:50:20.624720+00:00 [running]> on host 95e735c85bd1
[2024-05-05 09:50:25,500] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_Scoring_Data_Engineering_Pipeline
AIRFLOW_CTX_TASK_ID=building_db
AIRFLOW_CTX_EXECUTION_DATE=2024-05-05T09:50:20.624720+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-05T09:50:20.624720+00:00
[2024-05-05 09:50:25,501] {logging_mixin.py:115} INFO - ./utils_output.db asd
[2024-05-05 09:50:25,501] {logging_mixin.py:115} INFO - DB Already Exists
[2024-05-05 09:50:25,501] {python.py:173} INFO - Done. Returned value was: DB Exists
[2024-05-05 09:50:25,528] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=building_db, execution_date=20240505T095020, start_date=20240505T095025, end_date=20240505T095025
[2024-05-05 09:50:25,566] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-05-05 09:50:25,598] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
